_aggregations/pipeline/cumulative-sum.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+title: Cumulative sum
+parent: Pipeline aggregations
+has_children: false
+nav_order: 60
+---
+
+# Cumulative sum aggregations
+
+The `cumulative_sum` aggregation is a parent aggregation that calculates the cumulative sum of each bucket of a previous aggregation.
```

Comment:
Should the 2nd "of" be "in" or "from"?

Reply:
We can change to "across the buckets of a previous aggregation" to be more precise.